### PR TITLE
Convert SwitchTeamMessage to work with player's ID instead of username. 

### PR DESCRIPTION
**Pull request by BeeHiveJava**

### CODE REVIEW

File: Soccar [Game-Server]/src/nl/soccar/gameserver/message/SwitchTeamMessage.java [hunk 1]
```diff
@@ -13,21 +13,21 @@
...
 public final class SwitchTeamMessage extends Message {
 
-    private final String username;
+    private final int playerId;
     private final TeamColour team;
-    
-    public SwitchTeamMessage(String username, TeamColour team) {
-        this.username = username;
+
+    public SwitchTeamMessage(int playerId, TeamColour team) {
```
Comment (ghost) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/message/SwitchTeamMessage.java:19]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [Game-Server]/src/nl/soccar/gameserver/message/SwitchTeamMessage.java [hunk 2]
```diff
@@ -13,21 +13,21 @@
...
+        this.playerId = playerId;
         this.team = team;
     }
     
     public SwitchTeamMessage() {
```
Comment (ghost) [inline Soccar [Game-Server]/src/nl/soccar/gameserver/message/SwitchTeamMessage.java:24]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 3 issues
* MINOR 3 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [SessionWrapper.java#L110](https://github.com/PTS3-S34A/Game-Server/blob/20748c73a41612cdec1243c9d3ef199e7d552672/Soccar [Game-Server]/src/nl/soccar/gameserver/SessionWrapper.java#L110): Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by ghost** [comment]
SonarQube analysis reported 3 issues
* MINOR 3 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [SessionWrapper.java#L110](https://github.com/PTS3-S34A/Game-Server/blob/a41a2dba77c4aa02ec4505680bb8b2ec21f3a352/Soccar [Game-Server]/src/nl/soccar/gameserver/SessionWrapper.java#L110): Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Review by Ranray — APPROVED** [review]

**Pull request merged**